### PR TITLE
Fix become a lessor bug

### PR DESCRIPTION
**Pull request by neungkl**

### CODE REVIEW

**Comment by neungkl** [comment]
Please review and merge this commit @sirinthra-cc

File: webserver/apps/lessors/serializers.py [hunk 1]
```diff
@@ -37,17 +34,12 @@ def update(self, instance, validated_data):
...
 
     def validate(self, data):
-        try:
-            Lessor.objects.get(user=data['user'])
-        except ObjectDoesNotExist:
-            raise serializers.ValidationError('This user already registered as lessor')
-
-        if data['is_organization'] == True:
+        if data['is_organization']:
             if data['organization_name'] is None:
```
Comment (sirinthra-cc) [inline webserver/apps/lessors/serializers.py]: Check data['organization_name'] == '' instead

Comment (neungkl) [reply webserver/apps/lessors/serializers.py]: Fixed, please check again

**Comment by neungkl** [comment]
Fixed it. @sirinthra-cc

**Pull request merged**